src/Aspire.Dashboard/dashpages.json

```diff
@@ -0,0 +1,38 @@
+[
+  /* Comments are ignored */
```

Comment:
What do you think about documenting this schema elsewhere, and removing the comments? It is a good practice to avoid commenting JSON, even though our deserializer currently handles it.

Reply:
Comments were just because I copied this from a unit test where I wanted to validate that comments are tolerated.

Adding a schema makes sense.

Reply:
I do not think we should officially support JSON with comments. It is not valid JSON.

Reply:
It's very common to want to document your configuration. It's not very friendly to disallow that when it's so easy to permit it. I see no downside.

Reply:
For example, the .NET Project System ignores comments in `launchSettings.json` files.